WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -302,19 +304,42 @@ public static void openEditorForSiteInNewStack(Context context, @NonNull SiteMod
      * @param site     the site on which the post should be reblogged
      * @param post     the post to be reblogged
      */
-    public static void openEditorForReblog(Activity activity, @NonNull SiteModel site, @Nullable ReaderPost post) {
+    public static void openEditorForReblog(
+            Activity activity,
+            @Nullable SiteModel site,
+            @Nullable ReaderPost post,
+            PagePostCreationSourcesDetail reblogSource
+    ) {
         if (post == null) {
             ToastUtils.showToast(activity, R.string.post_not_found, ToastUtils.Duration.SHORT);
             return;
         }
+
+        if (site == null) {
+            ToastUtils.showToast(activity, R.string.blog_not_found, ToastUtils.Duration.SHORT);
+            return;
+        }
+
+        Map<String, Object> properties = new HashMap<>();
+        properties.put("source_post_id", post.postId);
+        properties.put("source_blog_id", post.blogId);
+        properties.put("target_blog_id", site.getSiteId());
```

Comment:
the `AnalyticsUtils` class has some of the fields used here. Do you think it might make sense to create a utility method there and reuse some of the properties? Maybe it doesn't but I'd still consider extracting it there and using the constants instead of these strings.

Reply:
Good suggestion thanks 🙇‍♂️! I moved to the Utils but kept the "source_*" and "target_*" prefix to be more clear on the intended usage; hope it's fine.